tests/test_doc.py

```diff
@@ -145,7 +220,10 @@ def run_doctest(root: str, filename: str, monkeypatch: pytest.MonkeyPatch):
         ):
             if safe_needed:
                 with SafeRunner():
-                    if "PYTEST_CURRENT_TEST" not in os.environ:
```

Comment:
this current_test thing was not useful right ?

Reply:
yes